src/DssSpell.sol

```diff
@@ -55,100 +63,81 @@ contract DssSpellAction is DssAction, DssSpellCollateralOnboardingAction {
     //
```

Comment:
Consider updating the table of rates IPFS hash also in `DssSpellCollateralOnboardingAction`

Reply:
Done!